1_The_Physical/Guess_Point_Level_1_Stateless/1_Red_Green_Refactor/1_1_Fizzbuzz/src/index.spec.ts

```diff
@@ -31,4 +31,8 @@ describe("fizzBuzz tests", () => {
   it("Knows that 5 is equal to 'Buzz'", () => {
     expect(fizzBuzz(5)).toBe("Buzz");
   });
+
+  it("Knows that 15 is equal to 'FizzBuzz'", () => {
```

Comment:
I like how short and straightforward your descriptions are, But this description was a little confusing to me at first because of what the term "equal" connotes. I would have phrased it as `"Knows that 15 will return 'FizzBuzz'"`